features/add_md_ref.feature

```diff
@@ -0,0 +1,109 @@
+@us40
```

Comment:
I'd have some consistency over the tags for the feature and the file name. Something we need to agree on but if the feature file name doesn't have the user story number then I wouldn't have it as a tag I would have what you have done previously using the description of the story as the tag i.e. @add_md_ref. Either this or change the file name to have the user story number at the beginning. Just helpful having this consistency for people who aren't as familiar with the tests.